firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/LoginsHelper.js

```diff
@@ -460,7 +460,7 @@ window.__firefox__.includeOnce("LoginsHelper", function() {
 
     const generatedPassword = PasswordGenerator.generatePassword({
       inputMaxLength: LoginManagerContent.activeField.maxLength,
-      rules: mapOfRules,
+      rules: mapOfRules ?? undefined,
```

Comment:
This will omit the prop all together when `mapOfRules` is falsy.
```suggestion
       ...(mapOfRules && { rules: mapOfRules }),
```